projects/1.20-fabric/assets/0-modrinth-mod/garnished/lang/zh_cn.json

```
@@ -140,6 +140,66 @@
   "item.garnished.incomplete_cashew_cookie": "未加工的腰果曲奇",
   "item.garnished.almond_cheese": "杏仁奶酪",
 
+  "item.garnished.peanut_cinder_sandwich": "淋花生油的余烬三明治",
+  "item.garnished.incomplete_peanut_cinder_sandwich": "未加工的淋花生油的余烬三明治",
```

```suggestion
  "item.garnished.incomplete_peanut_cinder_sandwich": "淋花生油的余烬三明治（半成品）",
```
官译里的`unprocessed`是前缀`未加工的`，`incomplete`是后缀`（半成品）`，请注意全文件修改

---

已修改（